pkg/basicstation/cups/server.go

```diff
@@ -293,13 +300,10 @@ func (s *Server) getTrust(address string) (*x509.Certificate, error) {
 		}
 		return nil, errNoTrust
 	}
-	_, host, port, err := parseAddress(address)
+	_, host, port, err := parseAddress("https:", address)
```

Comment:
I think you meant `https` without the colon

Reply:
Yup. Good catch.